app/Http/Controllers/Contacts/RelationshipsController.php

```diff
@@ -44,7 +45,7 @@ public function new(Request $request, Contact $contact)
         foreach (auth()->user()->account->relationshipTypes as $relationshipType) {
             $arrayRelationshipTypes->push([
                 'id' => $relationshipType->id,
-                'name' => $relationshipType->getLocalizedName($contact, true),
+                'label' => $relationshipType->getLocalizedName($contact, true),
```

Comment:
This broke the relationship dropdown 😀

![image](https://user-images.githubusercontent.com/61099/40696389-ad9c465c-6393-11e8-83e2-ff52bd0ec509.png)